packages/components/src/modal/index.js

```diff
@@ -108,14 +109,20 @@ class Modal extends Component {
 			children,
 			aria,
 			instanceId,
-			isDismissable,
+			isDismissible,
+			isDismissable, //Deprecated
 			// Many of the documented props for Modal are passed straight through
 			// to the ModalFrame component and handled there.
 			...otherProps
 		} = this.props;
 
 		const headingId = aria.labelledby || `components-modal-header-${ instanceId }`;
 
+		if ( isDismissable ) {
+			deprecated( 'isDismissable property', {
```

Comment:
Can we clarify a bit more here? Maybe: `isDismissable prop of the Modal component`

Reply:
surething